### PR TITLE
Quick Actions

### PR DESCRIPTION
**Pull request by prykon**
Add custom quick actions

### CODE REVIEW

**Comment by corsacca** [comment]
image
Thanks @prykon 
See the image for a couple suggestions.

**Comment by corsacca** [comment]
@prykon saving a new quick action doesn't seem to be working.

**Comment by prykon** [comment]
@corsacca Updates done, phpcs and all.

File: dt-core/admin/menu/tabs/tab-custom-lists.php [hunk 1]
```diff
@@ -567,7 +576,179 @@ private function process_languages_box() {
...
+                wp_die( 'Quick Action Update Error: Key is missing' );
+            } else {
+                // Add new Quick Action
+                $key = dt_create_field_key( $key, true );
+                $custom_field_options['contacts'][$key] = [
+                    'name'        => $label,
+                    'description' => '',
+                    'type'        => 'number',
+                    'default'     => 0,
+                    'section'     => 'quick_buttons_custom',
```
Comment (corsacca) [inline dt-core/admin/menu/tabs/tab-custom-lists.php:700]: @prykon is there a reason for the extra "_custom" here?

**Review by corsacca — APPROVED** [review]

**Pull request merged**